mina-daemon/templates/deployment.yaml

```diff
@@ -430,6 +429,11 @@ spec:
             - name: external-port
               protocol: TCP
               containerPort: {{ .Values.node.ports.p2p }}
+            {{- if .Values.node.exposeGraphql }}
+            - name: graphql-port
+              protocol: TCP
+              containerPort: {{ .Values.node.ports.graphql }}
+            {{- end -}}
```

Comment:
I am still leaving this block in case we want to access the raw graphql interface (without the public proxy in front of it)